README.md

```diff
@@ -27,7 +27,56 @@ These are the primary software versions used in production, and you should be ab
   - `brew doctor`
 
 
-### **2. Python**  
+### **2. (Mac) Install or update other dependencies:**
+
+```bash
+brew install libevent libmagic libxml2 libxslt openssl graphviz nginx
+brew install freetype libjpeg libtiff littlecms webp chromedriver
+brew tap petere/postgresql
+brew install postgresql@9.3
+brew link --force postgresql@9.3
+brew install --force node@6
+brew link node@6 --force
+brew cask install java8
+brew tap garrow/homebrew-elasticsearch17
+brew install elasticsearch@1.7
+pip3 install typing
+```
+>:star: _Note_: Elasticsearch 1.7 does not work with Java 9
+
+>:star: _Note_: To install Elasticsearch 5.x
+>- `brew search elasticsearch`
+>- `brew install elasticsearch@5.x`
```

Comment:
this can't actually be 5.x can it?   It would have to be 5.6?

Reply:
You're right. I will change this